auron-build.sh

```diff
@@ -54,8 +48,10 @@ MVN_CMD="$(dirname "$0")/build/mvn"
 PRE_PROFILE=false
 RELEASE_PROFILE=false
 CLEAN=true
+SKIP_TESTS=true
 SPARK_VER=""
 SCALA_VER=""
+JAVA_VER=""
```

Comment:
I think we can set the default javaver to 8.

I think javaver is different with scalaver and sparkver, the auron jar built on JDK8 can also run on JDK11/17.